pkg/spec/createconfig.go

```diff
@@ -0,0 +1,440 @@
+package createconfig
```

Comment:
This file should not be in here and should not be added back.  We have dropped support for pkg/spec